digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/controller/SyncController.java

```diff
@@ -19,22 +21,23 @@
 @RequestMapping("/sync/v1")
 public class SyncController {
 
-    private final SyncLogService syncLogService;
+    private final SyncService syncLogService;
```

Comment:
The reference name should be syncService

Reply:
Done